calculator.go

```diff
@@ -1,13 +1,120 @@
-// Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes a variable number of float64 and returns their sum.
+func Add(a float64, b float64, nums ...float64) float64 {
+	sum := a + b
+
+	for _, n := range nums {
+		sum += n
+	}
+
+	return sum
+}
+
+// Subtract takes a variable number of float64 and returns their difference.
+func Subtract(a float64, b float64, nums ...float64) float64 {
+	difference := a - b
+
+	for _, n := range nums {
+		difference -= n
+	}
+
+	return difference
+}
+
+// Multiply takes a variable number of float64 and returns their product.
+func Multiply(a float64, b float64, nums ...float64) float64 {
+	product := a * b
+
+	for _, n := range nums {
+		product *= n
+	}
+
+	return product
+}
+
+// Divide takes a variable number of float64 and returns their quotient, or an error if it's not a valid division.
```

Comment:
It's a good idea to wrap your comments at around 75 columns (you can use the `Rewrap` extension for VS Code to do this conveniently).

Reply:
Also, what's `valid`?

Reply:
In this case, just zero being the divisor. I've adjusted the comments accordingly!